examples/platform/silabs/efr32/rs911x/rsi_if.c

```diff
@@ -25,6 +25,7 @@
 #include "em_ldma.h"
 #include "em_usart.h"
 
+#include "FreeRTOS.h"
```

Comment:
Why is freeRTOS needed here if #include <cmsis_os2.h> is used lower?

cmsis should be favoured over freeRTOS since we are transitioning to full cmsis apis

Reply:
Good point, removed the FreeRTOS includes.